src/sass/include/_content.sass

```diff
@@ -92,7 +92,7 @@ body.protocol
   h2
     color: $tl-blue
     font-family: $family-secondary
-    font-weight: 300
+    font-weight: 500
```

Comment:
Why are we using `500` here while we use `600` on other websites for `h2` and the brand guidelines says 600 for h2?

Reply:
My mistake